README.MD

```diff
@@ -10,12 +10,34 @@ Emmie assists with creating unique and isolated environments for each feature be
 * Deploy all replication controllers and services to a template namespace. This can be a new namespace or configured to be "develop" branch
 
 ### Application Arguments:
-* listen-port: Port Emmie will listen on to take requests (NOTE: Only listents on HTTPS)
+* listen-port: Port Emmie will listen on to take requests (NOTE: Only listens on HTTPS)
 * docker-registry: Set to url of private docker registry
 * template-namespace: Namespace to 'clone from when creating new deployments'
 * path-to-tokens: Full path including file name to tokens file for authorization, setting to empty string will disable.
 
+## Building
+
+* Install Go
+  * Set your $GOPATH to something you like
+* Checkout or move this source directory to $GOPATH/src
+* Run `cd $GOPATH/src/emmie && make emmie` to get dependencies and build project
+* To build the docker immage, run `make`
+* To push your image to docker hub, set $PREFIX to your account name, then `make push`
+
+## Get Started
+1. Create auth tokens file (how?)
```

Comment:
Tokens can be whatever length or format you would like, I typically just generate a random string .